juriscraper/opinions/united_states/territories/pr.py

```diff
@@ -15,6 +16,7 @@
 class Site(OpinionSiteLinear):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        locale.setlocale(locale.LC_TIME, "es_ES.UTF-8")
```

Comment:
So I had this feeling that locale might be dangerous, and it seems like it probably is. Here's a SO post that says nasty stuff about it: 

https://stackoverflow.com/a/1551834/64911

In the docs, they also say things like:

> It is generally a bad idea to call [setlocale()](https://docs.python.org/3/library/locale.html#locale.setlocale) in some library routine, since as a side effect it affects the entire program

And:

> Saving and restoring it is almost as bad: it is expensive and affects other threads that happen to run before the settings have been restored.

I think we'll need another way of doing this...

Reply:
@mlissner I definitely noticed it affecting the whole program, hence setting it in abstract site. but that all makes sense.